tardis/montecarlo/packet_source.py

```diff
@@ -48,9 +49,9 @@ def create_uniform_packet_energies(no_of_packets, rng):
 
         Returns
         -------
-            energies for packets : numpy.ndarray
+            energies for packets : numpy.ndarray in Quantity units
         """
-        return np.ones(no_of_packets) / no_of_packets
+        return (np.ones(no_of_packets) / no_of_packets).to(u.erg)
```

Comment:
I would do `* u.erg` as the `.to` method is primarily used for unit conversions.